server/schemas/mutations/edit-expense.mutation.js

```diff
@@ -0,0 +1,39 @@
+import {
+  GraphQLInputObjectType,
+  GraphQLString,
+  GraphQLFloat,
+  GraphQLNonNull
+} from 'graphql';
+
+import CategoryType from '../types/category.type';
+import ExpenseType from '../types/expense.type';
+import Expense from '../../models/expense.model';
+
+const EditExpenseType = new GraphQLInputObjectType({
+  name: 'EditExpense',
+  fields: {
+    name: { type: GraphQLString },
+    category: { type: CategoryType },
+    amount: { type: GraphQLFloat },
+  },
+});
+
+export default {
+  type: new GraphQLNonNull(ExpenseType),
```

Comment:
These `non nulls`, is it possible to simply do an `ExpenseType!`, with `!` signifying required?

Reply:
Hmmm... 🤔

Reply:
Doesn't work 😢😞

Reply:
Haha. It's fine then. Used to do it that way while using graphene in python, that's why I was wondering.